### PR TITLE
Added fingerprint for Hyundai Elantra 2021

### PR DESCRIPTION
**Pull request by samuwh**
This fingerprint started popping up on my 2021 Hyundai Elantra SEL (non-hybrid).

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Which route did this come from?

**Comment by samuwh** [comment]
Hello,
5c9daa3b70d6f706|2021-09-16--17-24-13  is a route that had this fingerprint
show up (it was showing up as "mock").
5c9daa3b70d6f706|2021-09-16--18-22-58 is the route once I added fingerprint
to values.py (as shown in my pull request). It recognizes it fine now.

Please let me know if you need any more information! I'm new to this, I
just got my comma3 this week.

Sam

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -668,6 +668,7 @@ class Buttons:
       b'\xf1\x00HT6WA280BLHT6VA640A1CCN0N20NS5\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
       b'\xf1\x00HT6WA280BLHT6VA640A1CCN0N20NS5\x00\x00\x00\x00\x00\x00\xe8\xba\xce\xfa',
       b'\xf1\x87CXMQFM2135005JB2E\xb9\x89\x98W\xa9y\x97h\xa9\x98\x99wxvwh\x87\177\xffx\xff\xff\xff,,\xf1\x89HT6VA640A1\xf1\x82CCN0N20NS5\x00\x00\x00\x00\x00\x00',
+      b'\xf1\x87CXMQFM1916035JB2\x88vvgg\x87Wuwgev\xa9\x98\x88\x98h\x99\x9f\xffh\xff\xff\xff\xa5\xee\xf1\x89HT6VA640A1\xf1\x82CCN0N20NS5\000\000\000\000\000\000'
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:671]:
```suggestion
      b'\xf1\x87CXMQFM1916035JB2\x88vvgg\x87Wuwgev\xa9\x98\x88\x98h\x99\x9f\xffh\xff\xff\xff\xa5\xee\xf1\x89HT6VA640A1\xf1\x82CCN0N20NS5\x00\x00\x00\x00\x00\x00'
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -668,6 +668,7 @@ class Buttons:
...
     ],
     (Ecu.engine, 0x7e0, None): [b'\xf1\x82CNCWD0AMFCXCSFFA'],
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:673]:
```suggestion
    (Ecu.engine, 0x7e0, None): [
      b'\xf1\x82CNCWD0AMFCXCSFFA',
      b'\xf1\x82CNCWD0AMFCXCSFFB',
    ],
```

**Pull request merged**